Gems/Atom/Feature/Common/Assets/Shaders/SkyBox/SkyBox.azsl

```diff
@@ -174,6 +185,13 @@ PSOutput MainPS(VSOutput input)
     // Clamp lower bounds of sky to prevent zero or negative values, which can leads to NaNs in other shaders like SMAA
     color = max(0.000001f, color);
 
+#if ENABLE_MERGE_FILMIC_TONEMAP
+    // Apply manual exposure compensation
+    color *= pow(2.0, ViewSrg::GetExposureValueCompensation());
+    // Apply filmic tonemap
+    color = (color * (6.2f * color + 0.5f)) / (color * (6.2f * color + 1.7f)+ 0.06f);
```

Comment:
Here's that same math again, this should be factored out into a couple utility functions..